ghascompliance/__main__.py

```diff
@@ -33,6 +33,7 @@
 parser.add_argument("--disable-dependencies", action="store_true")
 parser.add_argument("--disable-secret-scanning", action="store_true")
 parser.add_argument("--is-github-app-token", action="store_true", default=False)
+parser.add_argument("--pr-comment", action="store_true", default=False)
```

Comment:
Do we want to enable this by default?

Reply:
I can see either way being acceptable. I wasn't sure if it was a feature everyone would want enabled by default.

Reply:
True, we can keep it opt-in.

Reply:
If it's enabled by default, wouldn't it make sense to change this flag to `--disable-pr-comment` similar to the flags for disabling specific technology checks?